### PR TITLE
Show views and fix some bugs

### PR DESCRIPTION
**Pull request by JoseRicardoJimenez**
# Description
What did you implemented/Why did you implemented this:
 - i've add show views
 - i've change the route path in the product card
 - i've fix footer
 - Let the reviewer know the overall effect of the PR.
 - Reference a ticket in your issue tracker. Explain what the change is and then and only then reference the ticket.
 - The “why” is a chance to explain both the engineering goal, but also a some business objective that is satisfied or moved along.
 
 
 # Testing
 Help me how can I test or look at the changes
 
 Example:
  - I've added coverage for testing all new methods. I used Faker for a few random user emails and names.
  
  ## Screenshots 
If applicable, include screenshots of the results or screenshots that help to see changes
Show_views

### CODE REVIEW

**Review by oramasangel — DISMISSED** [review]
great job

**Review by xFerchoVJ — APPROVED** [review]

**Review by OmarRoBra — APPROVED** [review]
Good Job :D

**Review by jairolan — APPROVED** [review]
Well done

File: app/views/customers/show.html.erb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<%= render 'layouts/navbar' %>
+
+<div>
+  <%= render 'see_more'%>    
+</div>
+
+<%= render 'layouts/footer' %>
```
Comment (YilKanda) [inline app/views/customers/show.html.erb:7]: Add a blank line

**Review by YilKanda — APPROVED** [review]

**Pull request merged**